meeting-notes/2024/2024-feb-01-meeting-notes.md

```diff
@@ -0,0 +1,41 @@
+# 2024 February 2 - Meeting notes
+
+## Meeting recording
+
+Watch the YouTube recording of the meeting: https://www.youtube.com/watch?v=1ZKq6ohi8Ko&list=PL9svBjLDUl_-gyXj2lDYomI8keXtbMqUl
+
+## Agenda and notes
+
+* **Introductions** - Tell us about yourself, where you're at in the world, how you use Salt and your experience level, why you are interested in this group, and what you hope to get out of it.
+- **How we work**
+  - Decision: We will record the meeting and let it be posted on YouTube.
+  - Decision: We will keep meeting notes in MD in a community repository.

```

Comment:
**nitpick (non-blocking):** better to be explicit since MD is an overloaded acronym

```suggestion
  - Decision: We will keep meeting notes in markdown in a community repository.

```

Reply:
I'll make this change in my draft rather commit yours. (If I commit yours, I have to pull it down and then also make my change and I prefer to just change it directly.)